source/ui/memory-component-presenter.cpp

```diff
@@ -195,23 +164,23 @@ MemoryComponentPresenter::dataInfo(const QModelIndex &index, int role) const {
 
 
 MemoryValue MemoryComponentPresenter::getMemoryValueCached(
-    const std::size_t memory_address, const std::size_t memory_length) const {
+    const std::size_t memoryAddress, const std::size_t memoryLength) const {
   // check for cache
-  if (_memoryCacheValid && memory_address >= _memoryCacheBaseAddress &&
-      memory_address + memory_length <=
+  if (_memoryCacheValid && memoryAddress >= _memoryCacheBaseAddress &&
+      memoryAddress + memoryLength <=
```

Comment:
Thanks for renaming the variables. Looks like noone reviewed these changes when they first came in :disappointed: